### ISSUE TITLE
Folders not loading in Chromium since update from bitwarden_rs to vaultwarden

### ISSUE THREAD

**Issue by murtuzaakhtari** [dani-garcia/vaultwarden#1790]
I updated a few weeks back from bitwarden_rs (on a Debian LXC container on Proxmox) to vaultwarden and also updated to the latest bw_web_build (v2.20.3). However, the folders do not load after logging into the web UI. Also the passwords do not load on Chromium until I click one of the Types (Login, Card, Identity, Secure Note). Once I click on the Types, the passwords show up but the folders still do not load. There are quite a few errors in the Dev Tools

I have since also upgraded to the latest bw_web_build (v2.20.4) but the issue persists.
The section for the folders and the passwords just keeps spinning:
![vaultwardenChromiumIssue](https://user-images.githubusercontent.com/723687/123046317-95c81a00-d3c1-11eb-84ae-231b78c80338.png)

Chromium Dev Tools error:
`WebSocketTransport.js:86 WebSocket connection to 'wss://bitwarden.myserver.com/notifications/hub?access_token=eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJuYmYiOjE2MjQ0MjkxNzcsImV4cCI6MTYyNDQzNjM3NywiaXNzIjoiaHR0cDovL2xvY2FsaG9zdHxsb2dpbiIsInN1YiI6IjI2MTA2ODriewD6tLVPEXcdzK0u6Oe72ZYYsTCizAs6YO6eIaCwOMOEyl3tYGUZ4D5vuQjTjdTvkaxC_-7LEsGYCmb2C55_PI1Jp6wrkVyQpljxuZLHwxBrdQO8jL9JoV2dctTC4EZwc6Z9KGcGyEdTUrLT-tQxXVN2ykIQ' failed: 
(anonymous) @ WebSocketTransport.js:86
t @ zone.js:913
(anonymous) @ WebSocketTransport.js:71
(anonymous) @ WebSocketTransport.js:33
(anonymous) @ WebSocketTransport.js:14
o @ WebSocketTransport.js:5
t.invoke @ zone.js:386
onInvoke @ core.js:28144
t.invoke @ zone.js:385
e.run @ zone.js:143
(anonymous) @ zone.js:891
t.invokeTask @ zone.js:421
onInvokeTask @ core.js:28132
t.invokeTask @ zone.js:420
e.runTask @ zone.js:188
m @ zone.js:601
Utils.js:198 [2021-06-23T06:19:37.411Z] Error: Failed to start the connection: Error: There was an error with the transport.
e.log @ Utils.js:198
(anonymous) @ HttpConnection.js:304
(anonymous) @ HttpConnection.js:33
(anonymous) @ HttpConnection.js:14
s @ HttpConnection.js:6
t.invoke @ zone.js:386
onInvoke @ core.js:28144
t.invoke @ zone.js:385
e.run @ zone.js:143
(anonymous) @ zone.js:891
t.invokeTask @ zone.js:421
onInvokeTask @ core.js:28132
t.invokeTask @ zone.js:420
e.runTask @ zone.js:188
m @ zone.js:601
e.invokeTask @ zone.js:507
p @ zone.js:1671
h @ zone.js:1697
error (async)
A @ zone.js:1823
t.scheduleTask @ zone.js:407
onScheduleTask @ zone.js:294
t.scheduleTask @ zone.js:400
e.scheduleTask @ zone.js:231
e.scheduleEventTask @ zone.js:257
(anonymous) @ zone.js:1980
E.i.set @ zone.js:1331
(anonymous) @ WebSocketTransport.js:98
t @ zone.js:913
(anonymous) @ WebSocketTransport.js:71
(anonymous) @ WebSocketTransport.js:33
(anonymous) @ WebSocketTransport.js:14
o @ WebSocketTransport.js:5
t.invoke @ zone.js:386
onInvoke @ core.js:28144
t.invoke @ zone.js:385
e.run @ zone.js:143
(anonymous) @ zone.js:891
t.invokeTask @ zone.js:421
onInvokeTask @ core.js:28132
t.invokeTask @ zone.js:420
e.runTask @ zone.js:188
m @ zone.js:601
Promise.then (async)
g @ zone.js:584
t.scheduleTask @ zone.js:410
onScheduleTask @ zone.js:294
t.scheduleTask @ zone.js:400
e.scheduleTask @ zone.js:231
e.scheduleMicroTask @ zone.js:251
E @ zone.js:881
w @ zone.js:819
(anonymous) @ zone.js:739
Promise.then (async)
(anonymous) @ zone.js:1110
t @ zone.js:913
D.t.then @ zone.js:1110
w @ zone.js:788
(anonymous) @ zone.js:739
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
l @ main.6318239ca8b236e2e537.js:1
a @ main.6318239ca8b236e2e537.js:1
t.invoke @ zone.js:386
onInvoke @ core.js:28144
t.invoke @ zone.js:385
e.run @ zone.js:143
(anonymous) @ zone.js:891
t.invokeTask @ zone.js:421
onInvokeTask @ core.js:28132
t.invokeTask @ zone.js:420
e.runTask @ zone.js:188
m @ zone.js:601
Promise.then (async)
g @ zone.js:584
t.scheduleTask @ zone.js:410
onScheduleTask @ zone.js:294
t.scheduleTask @ zone.js:400
e.scheduleTask @ zone.js:231
e.scheduleMicroTask @ zone.js:251
E @ zone.js:881
w @ zone.js:819
(anonymous) @ zone.js:739
Promise.then (async)
(anonymous) @ zone.js:1110
t @ zone.js:913
D.t.then @ zone.js:1110
w @ zone.js:788
(anonymous) @ zone.js:739
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
l @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
r @ main.6318239ca8b236e2e537.js:1
e.postIdentityToken @ main.6318239ca8b236e2e537.js:1
(anonymous) @ auth.service.ts:306
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
a @ main.6318239ca8b236e2e537.js:1
t.invoke @ zone.js:386
onInvoke @ core.js:28144
t.invoke @ zone.js:385
e.run @ zone.js:143
(anonymous) @ zone.js:891
t.invokeTask @ zone.js:421
onInvokeTask @ core.js:28132
t.invokeTask @ zone.js:420
e.runTask @ zone.js:188
m @ zone.js:601
Promise.then (async)
g @ zone.js:584
t.scheduleTask @ zone.js:410
onScheduleTask @ zone.js:294
t.scheduleTask @ zone.js:400
e.scheduleTask @ zone.js:231
e.scheduleMicroTask @ zone.js:251
E @ zone.js:881
w @ zone.js:819
(anonymous) @ zone.js:739
Promise.then (async)
(anonymous) @ zone.js:1110
t @ zone.js:913
D.t.then @ zone.js:1110
w @ zone.js:788
(anonymous) @ zone.js:739
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
l @ main.6318239ca8b236e2e537.js:1
a @ main.6318239ca8b236e2e537.js:1
t.invoke @ zone.js:386
onInvoke @ core.js:28144
t.invoke @ zone.js:385
e.run @ zone.js:143
(anonymous) @ zone.js:891
t.invokeTask @ zone.js:421
onInvokeTask @ core.js:28132
t.invokeTask @ zone.js:420
e.runTask @ zone.js:188
m @ zone.js:601
Promise.then (async)
g @ zone.js:584
t.scheduleTask @ zone.js:410
onScheduleTask @ zone.js:294
t.scheduleTask @ zone.js:400
e.scheduleTask @ zone.js:231
e.scheduleMicroTask @ zone.js:251
E @ zone.js:881
w @ zone.js:819
(anonymous) @ zone.js:739
Promise.then (async)
(anonymous) @ zone.js:1110
t @ zone.js:913
D.t.then @ zone.js:1110
w @ zone.js:788
(anonymous) @ zone.js:739
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
l @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
r @ main.6318239ca8b236e2e537.js:1
e.pbkdf2 @ main.6318239ca8b236e2e537.js:1
(anonymous) @ crypto.service.ts:374
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
o @ main.6318239ca8b236e2e537.js:1
e.hashPassword @ main.6318239ca8b236e2e537.js:1
(anonymous) @ auth.service.ts:124
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
a @ main.6318239ca8b236e2e537.js:1
t.invoke @ zone.js:386
onInvoke @ core.js:28144
t.invoke @ zone.js:385
e.run @ zone.js:143
(anonymous) @ zone.js:891
t.invokeTask @ zone.js:421
onInvokeTask @ core.js:28132
t.invokeTask @ zone.js:420
e.runTask @ zone.js:188
m @ zone.js:601
Promise.then (async)
g @ zone.js:584
t.scheduleTask @ zone.js:410
onScheduleTask @ zone.js:294
t.scheduleTask @ zone.js:400
e.scheduleTask @ zone.js:231
e.scheduleMicroTask @ zone.js:251
E @ zone.js:881
w @ zone.js:819
(anonymous) @ zone.js:739
Promise.then (async)
(anonymous) @ zone.js:1110
t @ zone.js:913
D.t.then @ zone.js:1110
w @ zone.js:788
(anonymous) @ zone.js:739
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
l @ main.6318239ca8b236e2e537.js:1
a @ main.6318239ca8b236e2e537.js:1
t.invoke @ zone.js:386
onInvoke @ core.js:28144
t.invoke @ zone.js:385
e.run @ zone.js:143
(anonymous) @ zone.js:891
t.invokeTask @ zone.js:421
onInvokeTask @ core.js:28132
t.invokeTask @ zone.js:420
e.runTask @ zone.js:188
m @ zone.js:601
Promise.then (async)
g @ zone.js:584
t.scheduleTask @ zone.js:410
onScheduleTask @ zone.js:294
t.scheduleTask @ zone.js:400
e.scheduleTask @ zone.js:231
e.scheduleMicroTask @ zone.js:251
E @ zone.js:881
w @ zone.js:819
(anonymous) @ zone.js:739
Promise.then (async)
(anonymous) @ zone.js:1110
t @ zone.js:913
D.t.then @ zone.js:1110
w @ zone.js:788
(anonymous) @ zone.js:739
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
l @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
r @ main.6318239ca8b236e2e537.js:1
e.pbkdf2 @ main.6318239ca8b236e2e537.js:1
(anonymous) @ crypto.service.ts:320
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
o @ main.6318239ca8b236e2e537.js:1
e.makeKey @ main.6318239ca8b236e2e537.js:1
(anonymous) @ auth.service.ts:252
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
a @ main.6318239ca8b236e2e537.js:1
t.invoke @ zone.js:386
onInvoke @ core.js:28144
t.invoke @ zone.js:385
e.run @ zone.js:143
(anonymous) @ zone.js:891
t.invokeTask @ zone.js:421
onInvokeTask @ core.js:28132
t.invokeTask @ zone.js:420
e.runTask @ zone.js:188
m @ zone.js:601
Promise.then (async)
g @ zone.js:584
t.scheduleTask @ zone.js:410
onScheduleTask @ zone.js:294
t.scheduleTask @ zone.js:400
e.scheduleTask @ zone.js:231
e.scheduleMicroTask @ zone.js:251
E @ zone.js:881
w @ zone.js:819
(anonymous) @ zone.js:739
Promise.then (async)
(anonymous) @ zone.js:1110
t @ zone.js:913
D.t.then @ zone.js:1110
w @ zone.js:788
(anonymous) @ zone.js:739
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
l @ main.6318239ca8b236e2e537.js:1
a @ main.6318239ca8b236e2e537.js:1
t.invoke @ zone.js:386
onInvoke @ core.js:28144
t.invoke @ zone.js:385
e.run @ zone.js:143
(anonymous) @ zone.js:891
t.invokeTask @ zone.js:421
onInvokeTask @ core.js:28132
t.invokeTask @ zone.js:420
e.runTask @ zone.js:188
m @ zone.js:601
Promise.then (async)
g @ zone.js:584
t.scheduleTask @ zone.js:410
onScheduleTask @ zone.js:294
t.scheduleTask @ zone.js:400
e.scheduleTask @ zone.js:231
e.scheduleMicroTask @ zone.js:251
E @ zone.js:881
w @ zone.js:819
(anonymous) @ zone.js:739
Promise.then (async)
(anonymous) @ zone.js:1110
t @ zone.js:913
D.t.then @ zone.js:1110
w @ zone.js:788
(anonymous) @ zone.js:739
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
l @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
r @ main.6318239ca8b236e2e537.js:1
e.send @ main.6318239ca8b236e2e537.js:1
(anonymous) @ api.service.ts:266
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
r @ main.6318239ca8b236e2e537.js:1
e.postPrelogin @ main.6318239ca8b236e2e537.js:1
(anonymous) @ auth.service.ts:242
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
i @ main.6318239ca8b236e2e537.js:1
e.makePreloginKey @ main.6318239ca8b236e2e537.js:1
(anonymous) @ auth.service.ts:123
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
i @ main.6318239ca8b236e2e537.js:1
e.logIn @ main.6318239ca8b236e2e537.js:1
(anonymous) @ login.component.ts:84
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
r @ main.6318239ca8b236e2e537.js:1
e.submit @ main.6318239ca8b236e2e537.js:1
(anonymous) @ login.component.html:1
vu @ core.js:14945
i @ core.js:14980
a @ core.js:25537
t.__tryOrUnsub @ Subscriber.js:192
t.next @ Subscriber.js:130
t._next @ Subscriber.js:76
t.next @ Subscriber.js:53
t.next @ Subject.js:47
t.emit @ core.js:25498
t.onSubmit @ forms.js:4776
(anonymous) @ forms.js:4817
vu @ core.js:14945
i @ core.js:14980
(anonymous) @ platform-browser.js:615
t.invokeTask @ zone.js:421
onInvokeTask @ core.js:28132
t.invokeTask @ zone.js:420
e.runTask @ zone.js:188
e.invokeTask @ zone.js:503
p @ zone.js:1671
h @ zone.js:1697
Show 128 more frames
core.js:4127 ERROR Error: Uncaught (in promise): TypeError: object is not iterable (cannot read property Symbol(Symbol.iterator))
TypeError: object is not iterable (cannot read property Symbol(Symbol.iterator))
    at new Set (<anonymous>)
    at t.<anonymous> (groupings.component.ts:65)
    at main.6318239ca8b236e2e537.js:1
    at Object.next (main.6318239ca8b236e2e537.js:1)
    at a (main.6318239ca8b236e2e537.js:1)
    at t.invoke (zone.js:386)
    at Object.onInvoke (core.js:28144)
    at t.invoke (zone.js:385)
    at e.run (zone.js:143)
    at zone.js:891
    at w (zone.js:832)
    at zone.js:739
    at s (main.6318239ca8b236e2e537.js:1)
    at t.invoke (zone.js:386)
    at Object.onInvoke (core.js:28144)
    at t.invoke (zone.js:385)
    at e.run (zone.js:143)
    at zone.js:891
    at t.invokeTask (zone.js:421)
    at Object.onInvokeTask (core.js:28132)
Ar @ core.js:4127
e.handleError @ core.js:4175
next @ core.js:28702
a @ core.js:25516
t.__tryOrUnsub @ Subscriber.js:192
t.next @ Subscriber.js:130
t._next @ Subscriber.js:76
t.next @ Subscriber.js:53
t.next @ Subject.js:47
t.emit @ core.js:25498
(anonymous) @ core.js:28167
t.invoke @ zone.js:386
e.run @ zone.js:143
e.runOutsideAngular @ core.js:28084
onHandleError @ core.js:28167
t.handleError @ zone.js:390
e.runGuarded @ zone.js:157
t @ zone.js:701
Zone.__load_patch.n.microtaskDrainDone @ zone.js:708
m @ zone.js:608
Promise.then (async)
g @ zone.js:584
t.scheduleTask @ zone.js:410
onScheduleTask @ zone.js:294
t.scheduleTask @ zone.js:400
e.scheduleTask @ zone.js:231
e.scheduleMicroTask @ zone.js:251
E @ zone.js:881
w @ zone.js:819
(anonymous) @ zone.js:739
Promise.then (async)
(anonymous) @ zone.js:1110
t @ zone.js:913
D.t.then @ zone.js:1110
w @ zone.js:788
(anonymous) @ zone.js:739
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
l @ main.6318239ca8b236e2e537.js:1
a @ main.6318239ca8b236e2e537.js:1
t.invoke @ zone.js:386
onInvoke @ core.js:28144
t.invoke @ zone.js:385
e.run @ zone.js:143
(anonymous) @ zone.js:891
t.invokeTask @ zone.js:421
onInvokeTask @ core.js:28132
t.invokeTask @ zone.js:420
e.runTask @ zone.js:188
m @ zone.js:601
Promise.then (async)
g @ zone.js:584
t.scheduleTask @ zone.js:410
onScheduleTask @ zone.js:294
t.scheduleTask @ zone.js:400
e.scheduleTask @ zone.js:231
e.scheduleMicroTask @ zone.js:251
E @ zone.js:881
w @ zone.js:819
(anonymous) @ zone.js:739
Promise.then (async)
(anonymous) @ zone.js:1110
t @ zone.js:913
D.t.then @ zone.js:1110
w @ zone.js:788
(anonymous) @ zone.js:739
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
l @ main.6318239ca8b236e2e537.js:1
a @ main.6318239ca8b236e2e537.js:1
t.invoke @ zone.js:386
onInvoke @ core.js:28144
t.invoke @ zone.js:385
e.run @ zone.js:143
(anonymous) @ zone.js:891
t.invokeTask @ zone.js:421
onInvokeTask @ core.js:28132
t.invokeTask @ zone.js:420
e.runTask @ zone.js:188
m @ zone.js:601
Promise.then (async)
g @ zone.js:584
t.scheduleTask @ zone.js:410
onScheduleTask @ zone.js:294
t.scheduleTask @ zone.js:400
e.scheduleTask @ zone.js:231
e.scheduleMicroTask @ zone.js:251
E @ zone.js:881
w @ zone.js:819
(anonymous) @ zone.js:739
Promise.then (async)
(anonymous) @ zone.js:1110
t @ zone.js:913
D.t.then @ zone.js:1110
w @ zone.js:788
(anonymous) @ zone.js:739
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
l @ main.6318239ca8b236e2e537.js:1
a @ main.6318239ca8b236e2e537.js:1
t.invoke @ zone.js:386
onInvoke @ core.js:28144
t.invoke @ zone.js:385
e.run @ zone.js:143
(anonymous) @ zone.js:891
t.invokeTask @ zone.js:421
onInvokeTask @ core.js:28132
t.invokeTask @ zone.js:420
e.runTask @ zone.js:188
m @ zone.js:601
Promise.then (async)
g @ zone.js:584
t.scheduleTask @ zone.js:410
onScheduleTask @ zone.js:294
t.scheduleTask @ zone.js:400
e.scheduleTask @ zone.js:231
e.scheduleMicroTask @ zone.js:251
E @ zone.js:881
w @ zone.js:819
(anonymous) @ zone.js:739
Promise.then (async)
(anonymous) @ zone.js:1110
t @ zone.js:913
D.t.then @ zone.js:1110
w @ zone.js:788
(anonymous) @ zone.js:739
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
l @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
r @ main.6318239ca8b236e2e537.js:1
e.postIdentityToken @ main.6318239ca8b236e2e537.js:1
(anonymous) @ auth.service.ts:306
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
a @ main.6318239ca8b236e2e537.js:1
t.invoke @ zone.js:386
onInvoke @ core.js:28144
t.invoke @ zone.js:385
e.run @ zone.js:143
(anonymous) @ zone.js:891
t.invokeTask @ zone.js:421
onInvokeTask @ core.js:28132
t.invokeTask @ zone.js:420
e.runTask @ zone.js:188
m @ zone.js:601
Promise.then (async)
g @ zone.js:584
t.scheduleTask @ zone.js:410
onScheduleTask @ zone.js:294
t.scheduleTask @ zone.js:400
e.scheduleTask @ zone.js:231
e.scheduleMicroTask @ zone.js:251
E @ zone.js:881
w @ zone.js:819
(anonymous) @ zone.js:739
Promise.then (async)
(anonymous) @ zone.js:1110
t @ zone.js:913
D.t.then @ zone.js:1110
w @ zone.js:788
(anonymous) @ zone.js:739
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
l @ main.6318239ca8b236e2e537.js:1
a @ main.6318239ca8b236e2e537.js:1
t.invoke @ zone.js:386
onInvoke @ core.js:28144
t.invoke @ zone.js:385
e.run @ zone.js:143
(anonymous) @ zone.js:891
t.invokeTask @ zone.js:421
onInvokeTask @ core.js:28132
t.invokeTask @ zone.js:420
e.runTask @ zone.js:188
m @ zone.js:601
Promise.then (async)
g @ zone.js:584
t.scheduleTask @ zone.js:410
onScheduleTask @ zone.js:294
t.scheduleTask @ zone.js:400
e.scheduleTask @ zone.js:231
e.scheduleMicroTask @ zone.js:251
E @ zone.js:881
w @ zone.js:819
(anonymous) @ zone.js:739
Promise.then (async)
(anonymous) @ zone.js:1110
t @ zone.js:913
D.t.then @ zone.js:1110
w @ zone.js:788
(anonymous) @ zone.js:739
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
l @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
r @ main.6318239ca8b236e2e537.js:1
e.pbkdf2 @ main.6318239ca8b236e2e537.js:1
(anonymous) @ crypto.service.ts:374
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
o @ main.6318239ca8b236e2e537.js:1
e.hashPassword @ main.6318239ca8b236e2e537.js:1
(anonymous) @ auth.service.ts:124
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
a @ main.6318239ca8b236e2e537.js:1
t.invoke @ zone.js:386
onInvoke @ core.js:28144
t.invoke @ zone.js:385
e.run @ zone.js:143
(anonymous) @ zone.js:891
t.invokeTask @ zone.js:421
onInvokeTask @ core.js:28132
t.invokeTask @ zone.js:420
e.runTask @ zone.js:188
m @ zone.js:601
Promise.then (async)
g @ zone.js:584
t.scheduleTask @ zone.js:410
onScheduleTask @ zone.js:294
t.scheduleTask @ zone.js:400
e.scheduleTask @ zone.js:231
e.scheduleMicroTask @ zone.js:251
E @ zone.js:881
w @ zone.js:819
(anonymous) @ zone.js:739
Promise.then (async)
(anonymous) @ zone.js:1110
t @ zone.js:913
D.t.then @ zone.js:1110
w @ zone.js:788
(anonymous) @ zone.js:739
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
l @ main.6318239ca8b236e2e537.js:1
a @ main.6318239ca8b236e2e537.js:1
t.invoke @ zone.js:386
onInvoke @ core.js:28144
t.invoke @ zone.js:385
e.run @ zone.js:143
(anonymous) @ zone.js:891
t.invokeTask @ zone.js:421
onInvokeTask @ core.js:28132
t.invokeTask @ zone.js:420
e.runTask @ zone.js:188
m @ zone.js:601
Promise.then (async)
g @ zone.js:584
t.scheduleTask @ zone.js:410
onScheduleTask @ zone.js:294
t.scheduleTask @ zone.js:400
e.scheduleTask @ zone.js:231
e.scheduleMicroTask @ zone.js:251
E @ zone.js:881
w @ zone.js:819
(anonymous) @ zone.js:739
Promise.then (async)
(anonymous) @ zone.js:1110
t @ zone.js:913
D.t.then @ zone.js:1110
w @ zone.js:788
(anonymous) @ zone.js:739
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
l @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
r @ main.6318239ca8b236e2e537.js:1
e.pbkdf2 @ main.6318239ca8b236e2e537.js:1
(anonymous) @ crypto.service.ts:320
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
o @ main.6318239ca8b236e2e537.js:1
e.makeKey @ main.6318239ca8b236e2e537.js:1
(anonymous) @ auth.service.ts:252
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
a @ main.6318239ca8b236e2e537.js:1
t.invoke @ zone.js:386
onInvoke @ core.js:28144
t.invoke @ zone.js:385
e.run @ zone.js:143
(anonymous) @ zone.js:891
t.invokeTask @ zone.js:421
onInvokeTask @ core.js:28132
t.invokeTask @ zone.js:420
e.runTask @ zone.js:188
m @ zone.js:601
Promise.then (async)
g @ zone.js:584
t.scheduleTask @ zone.js:410
onScheduleTask @ zone.js:294
t.scheduleTask @ zone.js:400
e.scheduleTask @ zone.js:231
e.scheduleMicroTask @ zone.js:251
E @ zone.js:881
w @ zone.js:819
(anonymous) @ zone.js:739
Promise.then (async)
(anonymous) @ zone.js:1110
t @ zone.js:913
D.t.then @ zone.js:1110
w @ zone.js:788
(anonymous) @ zone.js:739
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
l @ main.6318239ca8b236e2e537.js:1
a @ main.6318239ca8b236e2e537.js:1
t.invoke @ zone.js:386
onInvoke @ core.js:28144
t.invoke @ zone.js:385
e.run @ zone.js:143
(anonymous) @ zone.js:891
t.invokeTask @ zone.js:421
onInvokeTask @ core.js:28132
t.invokeTask @ zone.js:420
e.runTask @ zone.js:188
m @ zone.js:601
Promise.then (async)
g @ zone.js:584
t.scheduleTask @ zone.js:410
onScheduleTask @ zone.js:294
t.scheduleTask @ zone.js:400
e.scheduleTask @ zone.js:231
e.scheduleMicroTask @ zone.js:251
E @ zone.js:881
w @ zone.js:819
(anonymous) @ zone.js:739
Promise.then (async)
(anonymous) @ zone.js:1110
t @ zone.js:913
D.t.then @ zone.js:1110
w @ zone.js:788
(anonymous) @ zone.js:739
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
l @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
r @ main.6318239ca8b236e2e537.js:1
e.send @ main.6318239ca8b236e2e537.js:1
(anonymous) @ api.service.ts:266
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
r @ main.6318239ca8b236e2e537.js:1
e.postPrelogin @ main.6318239ca8b236e2e537.js:1
(anonymous) @ auth.service.ts:242
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
i @ main.6318239ca8b236e2e537.js:1
e.makePreloginKey @ main.6318239ca8b236e2e537.js:1
(anonymous) @ auth.service.ts:123
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
i @ main.6318239ca8b236e2e537.js:1
e.logIn @ main.6318239ca8b236e2e537.js:1
(anonymous) @ login.component.ts:84
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
(anonymous) @ main.6318239ca8b236e2e537.js:1
t @ zone.js:913
r @ main.6318239ca8b236e2e537.js:1
e.submit @ main.6318239ca8b236e2e537.js:1
(anonymous) @ login.component.html:1
vu @ core.js:14945
i @ core.js:14980
a @ core.js:25537
t.__tryOrUnsub @ Subscriber.js:192
t.next @ Subscriber.js:130
t._next @ Subscriber.js:76
t.next @ Subscriber.js:53
t.next @ Subject.js:47
t.emit @ core.js:25498
t.onSubmit @ forms.js:4776
(anonymous) @ forms.js:4817
vu @ core.js:14945
i @ core.js:14980
(anonymous) @ platform-browser.js:615
t.invokeTask @ zone.js:421
onInvokeTask @ core.js:28132
t.invokeTask @ zone.js:420
e.runTask @ zone.js:188
e.invokeTask @ zone.js:503
p @ zone.js:1671
h @ zone.js:1697
Show 128 more frames`

Firefox has no problem displaying the folders and the passwords -- however Dev Tools on Firefox also shows the same errors, so I am not sure if the errors are related to the issue that I have with Chromium.

Here's the error from the Firefox Dev Tools:
`GETwss://bitwarden.myserver.com/notifications/hub?access_token=eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJuYmYiOjE2MjQ0MjkxNzcsImV4cCI6MTYyNDQzNjM3NywiaXNzIjoiaHR0cDovL2xvY2FsaG9zdHxsb2dpbiIsInN1YiI6IjI2MTA2ODriewD6tLVPEXcdzK0u6Oe72ZYYsTCizAs6YO6eIaCwOMOEyl3tYGUZ4D5vuQjTjdTvkaxC_-7LEsGYCmb2C55_PI1Jp6wrkVyQpljxuZLHwxBrdQO8jL9JoV2dctTC4EZwc6Z9KGcGyEdTUrLT-tQxXVN2ykIQ
[HTTP/1.1 400 Bad Request 18ms]

Firefox can’t establish a connection to the server at wss://bitwarden.myserver.com/notifications/hub?access_token=eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJuYmYiOjE2MjQ0MjkxNzcsImV4cCI6MTYyNDQzNjM3NywiaXNzIjoiaHR0cDovL2xvY2FsaG9zdHxsb2dpbiIsInN1YiI6IjI2MTA2ODriewD6tLVPEXcdzK0u6Oe72ZYYsTCizAs6YO6eIaCwOMOEyl3tYGUZ4D5vuQjTjdTvkaxC_-7LEsGYCmb2C55_PI1Jp6wrkVyQpljxuZLHwxBrdQO8jL9JoV2dctTC4EZwc6Z9KGcGyEdTUrLT-tQxXVN2ykIQ. WebSocketTransport.js:86:44
`

I am using HAProxy plugin (in OPNsense) as a reverse proxy with SSL off-loading to bitwarden but all that used to work just fine until I upgraded to vaultwarden.

I have not found anything on the web related to the issue that I am facing. Can someone please help me fix this?

**Comment by BlackDex** [comment]
Do you also have some caching enabled for Nginx? That could though in some issues.
If so, try to clear the nginx cache, also see what happens when using an incognito/private browser window.

**Comment by murtuzaakhtari** [comment]
> Do you also have some caching enabled for Nginx? That could cause some issues.
> If so, try to clear the nginx cache, also see what happens when using an incognito/private browser window.

Thanks for the quick reply. I tried out the Incognito, and it worked fine. So I created a new profile in Chromium and that worked fine as well. Is there a way to force update the cached files -- just so that I can avoid having to re-install the extensions/bookmarks etc in the new profile?

As for Nginx -- I am not using Nginx anywhere for bitwarden/vaultwarden. I use HAProxy plugin on OPNsense as a reverse proxy to the vaultwarden container.
